lib/internal/test_runner/coverage.js

```diff
@@ -172,7 +172,7 @@ class TestCoverage {
           }
         }
 
-        if (functionName.length > 0 && ranges.length > 0) {
+        if (j > 0 && ranges.length > 0) {
```

Comment:
is this also correct in the case of ESM?

Reply:
i have the same question; CJS is wrapped in a function but i don't expect ESM to be

Reply:
Yes. I thought maybe it was a CJS specific thing, but it's not.

Reply:
```
    {
      "scriptId": "422",
      "url": "file:///Users/cjihrig/iojs/node/test.mjs",
      "functions": [
        {
          "functionName": "",
          "ranges": [
            {
              "startOffset": 0,
              "endOffset": 4462,
              "count": 1
            }
          ],
          "isBlockCoverage": true
        },
        {
          "functionName": "chickenSandwich",
          "ranges": [
            {
              "startOffset": 49,
              "endOffset": 80,
              "count": 1
            }
          ],
          "isBlockCoverage": true
        }
      ]
    },
```

Reply:
weird, is that something added by v8, or by node's esm implementation?

Reply:
That part is unclear to me.